### DISCUSSION TITLE
Get email from shared email

### DISCUSSION THREAD

**Discussion by azreal1231** [O365/python-o365#784] [Q&A]
Hi, ive got an authed app but now im trying to access emails from a shared email. this is what ive implemented atm.

email_add = 'the-shared-email-address'

credentials = (APPLICATION_ID, CLIENT_SECRET)
account = Account(credentials=credentials, scopes=scopes)
mailbox = account.mailbox(resource=email_add)
inbox = mailbox.get_folder(folder_name='Inbox')
for message in inbox.get_messages(5):
    print(message)
    print('-' * 50)

-----------------------------------ERROR-------------------------------------------------
Error Message: The specified object was not found in the store., The process failed to get the correct properties.

**Comment by alejcas** [comment]
I've never before see that error message.
